modules/programs/tmux.nix

```
@@ -204,6 +205,14 @@ in {
         description = "VI or Emacs style shortcuts.";
       };
 
+      mouse = mkOption {
+        default = false;
+        example = true;
+        description = ''
+          Mouse support.
+        '';
+      };
```

This can be simplified:
```suggestion
      mouse = mkEnableOption "mouse support";
```

---

Ah this is much nicer, thanks!